templates/taobao-mini-game/game.ejs

```diff
@@ -0,0 +1,67 @@
+loadCC();
+
+function loadCC() {
+
+    require('./web-adapter');
+
+<% if(polyfillsBundleFile) { %>
+    // Polyfills bundle.
+    require("<%= polyfillsBundleFile %>");
+<% } %>
+
+    // SystemJS support.
+    require("<%= systemJsBundleFile %>");
+
+
+    // Adapt for IOS, swap if opposite
+    if (canvas){
+        var _w = canvas.width;
+        var _h = canvas.height;
+        if (screen.width < screen.height) {
+            if (canvas.width > canvas.height) {
+                _w = canvas.height;
+                _h = canvas.width;
+            }
+        } else {
+            if (canvas.width < canvas.height) {
+                _w = canvas.height;
+                _h = canvas.width;
+            }
+        }
+        canvas.width = _w;
+        canvas.height = _h;
+    }
+
+    // Adjust initial canvas size
+    if (canvas && window.devicePixelRatio >= 2) {canvas.width *= 2; canvas.height *= 2;}
```

Comment:
do we need these for taobao iOS ?
can we remove these ?

Reply:
Need to test to verify

Reply:
We currently have no testable apps

Reply:
then remember to update this part when you get testable app